content/zh-cn/blog/falco-0-28-0.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Falco 0.28.0 a.k.a. Falco 2021.04
+date: 2021-04-09
+author: Leonardo Di Donato
+slug: falco-0-28-0
+---
+
+今天我们宣布春季发布Falco 0.28.0
+
+这是Falco在2021年的第二次发布！
+
+您可以在此处查看一组更改：
+
+- 0.28.0 https://github.com/falcosecurity/falco/releases/tag/0.28.0
```

Comment:
This can be rewritten as a Markdown link: https://spec-md.com/#sec-Links
i.e. [0.28.0](https://github.com/falcosecurity/falco/releases/tag/0.28.0)